osu.Game.Rulesets.Catch/Objects/Drawables/DrawablePalpableCatchHitObject.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Collections.Generic;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osuTK;
+using osuTK.Graphics;
+
+namespace osu.Game.Rulesets.Catch.Objects.Drawables
+{
+    public abstract class DrawablePalpableCatchHitObject : DrawableCatchHitObject
+    {
+        public virtual bool StaysOnPlate => true;
```

Comment:
This is overriden, but never actually used anywhere. Should just stick with the property on the HO, probably.

Reply:
I thought it should belong to `Drawable*` because it is a "visual only" property. It won't affect autoplay generation or difficulty calculation for example.